Packs/MISPFeed/Integrations/MISPFeed/MISPFeed.py

```diff
@@ -0,0 +1,527 @@
+        FeedIndicatorType.URL: EntityRelationship.Relationships.INDICATOR_OF,
+        FeedIndicatorType.Email: EntityRelationship.Relationships.INDICATOR_OF,
+        FeedIndicatorType.Registry: EntityRelationship.Relationships.RELATED_TO,
+        ThreatIntel.ObjectsNames.THREAT_ACTOR: EntityRelationship.Relationships.RELATED_TO,
+        DBotScoreType.CRYPTOCURRENCY: EntityRelationship.Relationships.RELATED_TO,
+        ThreatIntel.ObjectsNames.MALWARE: EntityRelationship.Relationships.RELATED_TO,
+        ThreatIntel.ObjectsNames.CAMPAIGN: EntityRelationship.Relationships.ATTRIBUTED_TO,
+    },
+    ThreatIntel.ObjectsNames.COURSE_OF_ACTION: {
+        FeedIndicatorType.File: EntityRelationship.Relationships.RELATED_TO,
+        FeedIndicatorType.IP: EntityRelationship.Relationships.RELATED_TO,
+        FeedIndicatorType.Domain: EntityRelationship.Relationships.RELATED_TO,
+        FeedIndicatorType.URL: EntityRelationship.Relationships.RELATED_TO,
+        FeedIndicatorType.Email: EntityRelationship.Relationships.RELATED_TO,
+        FeedIndicatorType.Registry: EntityRelationship.Relationships.RELATED_TO,
+        ThreatIntel.ObjectsNames.THREAT_ACTOR: EntityRelationship.Relationships.RELATED_TO,
+        DBotScoreType.CRYPTOCURRENCY: EntityRelationship.Relationships.RELATED_TO,
+        ThreatIntel.ObjectsNames.MALWARE: EntityRelationship.Relationships.MITIGATED_BY,
+        ThreatIntel.ObjectsNames.CAMPAIGN: EntityRelationship.Relationships.RELATED_TO,
+    }
+}
+
+""" Client Class """
+
+
+class Client(BaseClient):
+    def search_query(self, body: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Creates a request to MISP to get all attributes filtered by query in the body argument
+        Args:
+            body: Dictionary containing query to filter MISP attributes.
+        Returns: bytes representing the response from MISP API
+        """
+        headers = {
+            'Authorization': demisto.params().get('apikey'),
+            "Accept": "application/json",
+            'Content-Type': 'application/json'
+        }
+        response = self._http_request('POST',
+                                      full_url=f'{self._base_url}attributes/restSearch',
+                                      resp_type='json',
+                                      headers=headers,
+                                      data=json.dumps(body))
+        return response
+
+
+""" Helper Functions """
+
+
+def build_indicators_iterator(attributes: Dict[str, Any], url: Optional[str]) -> List[Dict[str, Any]]:
+    """
+    Creates a list of valid indicators types to be created
+    Args:
+        attributes: List of attributes returned from MISP
+        url: Feed URL
+    Returns: List of indicators and their types
+    """
+    indicators_iterator = []
+    try:
+        attributes_list: List[Dict[str, Any]] = attributes['response']['Attribute']
+        for attribute in attributes_list:
+            if get_attribute_indicator_type(attribute):
```

Comment:
If I do 
`attributes_list := attributes['response']['Attribute']`
I get _Local variable 'attributes_list' value is not used _
And I don't want to do `for attribute in attributes['response']['Attribute']:` because it's not clear what's the content of this field.
Maybe I should just do `for attribute in attributes['response']['Attribute']:` and comment.

What is your preference?

Reply:
I meant that you can do `if indicator_type := get_attribute_indicator_type(attribute):` and then you can update the type in line 150 to be `indicator_type`

Reply:
fixed